source/whats-new.txt

```diff
@@ -102,6 +102,17 @@ What's New in 4.0
      library equivalents. For context, see the :ref:`Build System Breaking Changes
      <version-4.0-build-system-breaking>` admonition.
 
+   - Removes the following deprecated ``utf8`` types that have ``string`` equivalents:
```

Comment:
S: you could consider removing the full list from here and just linking to the upgrade guide for specific details and keep the list there?